app/javascript/controllers/campaign_tag_filter_controller.js

```diff
@@ -0,0 +1,80 @@
+import {Controller} from "@hotwired/stimulus";
+
+export default class extends Controller {
+  static get targets() {
+    return ['selectedTagsDom', 'unselectedTagsDom', 'tagList']
+  }
+
+  connect() {
+    this.companyTags = this.element.dataset.companyTags.split(',').filter(item => !!item)
+    this.selectedTags = []
+
+    $(window).click(() => this.tagListTarget.classList.add('d-none'));
+    $(this.tagListTarget).click(event => event.stopPropagation());
+  }
+
+  displayList(event) {
+    event.stopPropagation()
+    this.tagListTarget.classList.remove('d-none')
+  }
+
+  toggleTag(e) {
```

Comment:
@harold93 This seems a bit complex. Possible to simply render a partial in html server-side and injecting it on the fetch response?

Reply:
it already have this behavior, I receive the partial "render campaigns/index/index_campaigns_list"
the code below only adds some styling